docs/docs/cmd/entra/pim/pim-role-assignment-add.mdx

```diff
@@ -0,0 +1,221 @@
+: Id of the role definition that is being assigned. Specify either `roleDefinitionName` or `roleDefinitionId` but not both.
+
+`--userId [userId]`	
+: Id of the user that will be granted the assignment. Specify either `userId`, `userName`, `groupId` or `groupName`. If not specified, the current user will be used.
+
+`--userName [userName]`	
+: UPN of the user that will be granted the assignment. Specify either `userId`, `userName`, `groupId` or `groupName`. If not specified, the current user will be used.
+
+`--groupId [groupId]`
+: Id of the group that will be granted the assignment. Specify either `userId`, `userName`, `groupId` or `groupName`. If not specified, the current user will be used.
+
+`--groupName [groupName]`
+: Display name of the group that will be granted the assignment. Specify either `userId`, `userName`, `groupId` or `groupName`. If not specified, the current user will be used.
+
+`--administrativeUnitId [administrativeUnitId]`
+: Id of the administrative unit representing the scope of the assignment. Specify either `administrativeUnitId` or `applicationId`. If not specified, default scope will be tenant-wide.
+
+`--applicationId [applicationId]`
+: Object id of the application representing the scope of the assignment. Specify either `administrativeUnitId` or `applicationId`. If not specified, default scope will be tenant-wide.
+
+`-j, --justification [justification]`
+: An optional justification message.
+
+`-s, --startDateTime [startDateTime]`
+: When the assignment should start. If left out, the assignment will start from the current time.
+
+`-e, --endDateTime [endDateTime]`
+: When the assignment should end. Specify either `duration` or `endDateTime`.
+
+`-d, --duration [duration]`
+: How long the assignment should last. Write in ISO 8601 format for durations: PT3H for 3 hours.
+
+`--ticketNumber [ticketNumber]`
+: Optional ticket number value to communicate with the request.
+
+`--ticketSystem [ticketSystem]`
+: Optional ticket system to communicate with the request.
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To use this command you must be **Privileged Role Administrator**.
+
+:::
+
+## Examples
+
+Request activation of the SharePoint Administrator Entra ID role assignment for the current user.
+
+```sh
+m365 entra pim role assignment add --roleDefinitionName 'SharePoint Administrator'
+```
+
+Request activation of an Entra ID role assignment for the current user.
+
+```sh
+m365 entra pim role assignment add --roleDefinitionId 'f1417aa3-bf0b-4cc5-a845-a0b2cf11f690'
+```
+
+Request activation of an Entra ID role assignment for the current user with a justification and max duration of 4 hours.
+
+```sh
+m365 entra pim role assignment add --roleDefinitionId 'f1417aa3-bf0b-4cc5-a845-a0b2cf11f690' --justification 'Need Global Admin to release application xyz to production' --duration 'PT4H'
+```
+
+Request activation of an Entra ID role assignment for a specified user with tenant scope.
```

Comment:
When referring to Entra ID as a product, we use the full name. When referring to a part of the product we need to skip the ID part. So:

"We work with Entra ID"
"We added an Entra Group"
"I requested activation of an Entra role assignment"